### ISSUE TITLE
Received URL without a pathname #logotype.

### ISSUE THREAD

**Issue by FRSgit** [parcel-bundler/parcel#5175]
# 🐛 bug report

Parcel fails when it stumbles across html fragment with svg sprite used like this:

```html
<svg xmlns:xlink="http://www.w3.org/1999/xlink">
   <use xlink:href="#logotype" href="#logotype"></use>
</svg>
```

## 🎛 Configuration (.babelrc, package.json, cli command)

Default configs, happens when running both `parcel` and `parcel build`.

```js
{
  "scripts": { "build": "parcel build src/index.html" }
}
```

## 🤔 Expected Behavior

The HTML should be bundled without an error.

## 😯 Current Behavior

Parcel throws an error:
![image](https://user-images.githubusercontent.com/10456649/94004561-2a530c80-fd9d-11ea-9544-55688a71a487.png)

## 💁 Possible Solution

Seems like there was already similar issue for CSS files (#3873), seems like there should be some kind of ignoring happening for urls that are beginning with a `#` sign - either it would be done automatically by parcel or by some `ignore-paths` configuration option.

## 🔦 Context

This bug basically makes it impossible to create SVG sprites with use of SVG `<use>` tag.

## 💻 Code Sample

Create new parcel project, paste the code from summary to the index.html and run `parcel` or `parcel build`.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.0.0-beta.1
| Node             | 10.21.0
| Yarn         | 1.22.4
| Operating System | MacOS

**Comment by mischnic** [comment]
We need to add a special case somewhere in here:

https://github.com/parcel-bundler/parcel/blob/v2/packages/transformers/html/src/dependencies.js#L121

And a test case here: https://github.com/parcel-bundler/parcel/blob/v2/packages/core/integration-tests/test/html.js

**Comment by FRSgit** [comment]
Closing as this was fixed by #5188.
